File: mpp/Lab8/csharpPb9/csharpPb9/model/Proba.cs
public class Proba : Entity<string>
{
    public string Nume { get; set; }
    public long Id_arbitru { get; set; }
    public Categorie Categorie { get; set; }

    public Proba(string id, string nume, Categorie categorie)
    {
        Id = id;
        Nume = nume;
        this.Categorie = categorie;
        Id_arbitru = -1;
    }

    public Proba(string id,string nume, Categorie categorie, long id_arbitru)
    {
        Id = id;
        Nume = nume;
        Categorie = categorie;
        Id_arbitru = id_arbitru;
    }

    public override string ToString()
    {
        return Id + " " + Nume;
    }

}

File: mpp/Lab12/csharpPb9/web_app/src/proba.ts
export interface Proba {
    id: string;
    nume: string;
    id_arbitru: number;
    categorie: 0 | 1 | 2;
}

export interface ProbaDTO{
    nume: string;
    categorie: string;
}

export interface Arbitru{
    id: number;
    nume: string;
    username : string;
}